python/lsst/ctrl/mpexec/mpGraphExecutor.py

```diff
@@ -451,11 +498,15 @@ def _executeQuantaMP(self, graph, butler):
                     _LOG.debug("finished: %s", job)
                     # finished
                     exitcode = job.process.exitcode
+                    report = job.report()
+                    if report:
+                        self.jobReport.quantaReports.append(report)
```

Comment:
I see that it can be confusing in this context. The Job class in this module is not exposed, it's just an implementation detail of this executor. JobReport class is what appears in QuantumGraphExecutor interface, so I don't think there will be a confusion about what it means.